server/src/test/java/org/eclipse/hono/TestRegistrationService.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (c) 2017 Bosch Software Innovations GmbH.
+ * <p>
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * <p>
+ * Contributors:
+ * Bosch Software Innovations GmbH - initial creation
+ */
+
+package org.eclipse.hono;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonObject;
+import org.eclipse.hono.service.registration.BaseRegistrationService;
+import org.eclipse.hono.util.RegistrationResult;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static java.net.HttpURLConnection.*;
+import static org.eclipse.hono.util.RegistrationConstants.*;
+
+/**
+ * A registration service that keeps all data in memory. Serves for test puposes only.
+ *
+ */
+public class TestRegistrationService extends BaseRegistrationService {
```

Comment:
While trying to mock it, it turned out that the registration service is not needed at all anymore for the standalone test.
Thus I will remove it instead of mocking it.